src/test/java/com/scriptbasic/executors/operators/TestOperators.java

```diff
@@ -175,6 +175,9 @@ public void testOperators() throws AnalysisException,
         a("a=1 div 1", 1);
         a("a= 1 div 3", 0);
         a("a=1.0 div 3.0", 0);
+        
+        a("a= \"x\"&\"y\" = \"xy\"", true);
+        a("a= \"x\"& 1 = \"x1\"", true);
```

Comment:
same

Reply:
I understand that you are not fan of the operator & neither am I. On the other hand the operator is part of the Basic. There exists argumentation for using this operator instead of plus. If either of the expressions is not a string type, it is converted to String and then concatenated.

I am trying to interpret real Basic code with jScriptBasic and this operator is used quite often. This is my main motivation for this proposal. 

Please add this feature to the jScriptBasic. It will allow to use JSB for interpreting existing Basic code.

Reply:
OK. I accept that. I will.

Reply:
Please also add test cases when the 

1. left
2. the right
3. both operands are undef